EIPS/eip-1.md

```diff
@@ -1,101 +1,100 @@
-* **Final** -- This EIP represents the current state-of-the-art. A Final EIP should only be updated to correct errata.
+Each status change is requested by the EEP author and reviewed by the EEP editors. Use a pull request to update the status. Please include a link to where people should continue discussing your EEP. The EEP editors will process these requests as per the conditions below.
+
+* **Work in progress (WIP)** -- Once the champion has asked the EOS community whether an idea has any chance of support, they will write a draft EEP as a [pull request]. Consider including an implementation if this will aid people in studying the EEP.
+  * :arrow_right: Draft -- If agreeable, EEP editor will assign the EEP a number (generally the issue or PR number related to the EEP) and merge your pull request. The EEP editor will not unreasonably deny an EEP.
+  * :x: Draft -- Reasons for denying draft status include being too unfocused, too broad, duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the [EOS philosophy]. [FLAG - WE DON'T HAVE AN EOS PHILOSOPHY TEXT LIKE ETH HAD SET UP]
+* **Draft** -- Once the first draft has been merged, you may submit follow-up pull requests with further changes to your draft until such point as you believe the EEP to be mature and ready to proceed to the next status. An EEP in draft status must be implemented to be considered for promotion to the next status (ignore this requirement for core EEPs).
+  * :arrow_right: Last Call -- If agreeable, the EEP editor will assign Last Call status and set a review end date, normally 14 days later.
+  * :x: Last Call -- A request for Last Call status will be denied if material changes are still expected to be made to the draft. We hope that EEPs only enter Last Call once, so as to avoid unnecessary noise on the RSS feed. Last Call will be denied if the implementation is not complete and supported by the community.
+* **Last Call** -- This EEP will listed prominently on the [NEED WEBSITE CREATED] website [NEED TO CREATE AN RSS] (subscribe via RSS at [last-call.xml](/last-call.xml)).
+  * :x: -- A Last Call which results in material changes or substantial unaddressed complaints will cause the EEP to revert to Draft.
+  * :arrow_right: Accepted (Core EEPs only) -- After the review end date, the [IF WE HAVE SOMETHING SIMILAR TO CORE DEVS] EOS Core Developers will vote on whether to accept this change. If yes, the status will upgrade to Accepted.
+  * :arrow_right: Final (Not core EEPs) -- A successful Last Call without material changes or unaddressed complaints will become Final.
+* **Accepted (Core EEPs only)** -- This is being implemented by [FLAGGING FOR CORE DEV] EOS Core Developers.
```

Comment:
That will be harder :) We might need to add some section in this document stating how exactly once accepted, the development should go.